api/challenge_api/README.md

```diff
@@ -0,0 +1,24 @@
+# README
```

Comment:
I'm not sure how useful this boilerplate file is. We should either (1) fill it out with relevant information on how to start, configure and use the API, (2) remove the file entirely and fill out the README.md at the root level of this repo or (3) remove this file until we have something meaningful to include in it.

Reply:
Cool, I wasn't sure about that. I'll delete it for now.